logger.go

```diff
@@ -11,11 +11,12 @@ import (
  * Base struct
  */
 type Logger struct {
-	transports []*Transport
+	transports  []*Transport
+	DefaultMeta map[string]string
```

Comment:
This is a public field. Unless you want this to be modified at runtime, make it private.